modules/base/tool.go

```diff
@@ -31,7 +31,16 @@ import (
 	"github.com/gogits/gogs/modules/setting"
 )
 
-var Sanitizer = bluemonday.UGCPolicy().AllowAttrs("class").Matching(regexp.MustCompile(`[\p{L}\p{N}\s\-_',:\[\]!\./\\\(\)&]*`)).OnElements("code")
+func BuildSanitizer() (p *bluemonday.Policy) {
```

Comment:
Because I didn't know how golang worked with line-breaks, so I hade a function of it instead. I can revert that, though #2406 needs to do that anyhow to get the config-flag.